lib/tools/broccoli/builder.ts

```diff
@@ -125,5 +131,11 @@ export class Builder implements IBroccoliBuilder {
 
 		}).future<void>()();
 	}
+
+	public cleanNodeModules(absoluteOutputPath: string, platform: string): IFuture<void> {
+		return (() => {
+			shelljs.rm("-rf", absoluteOutputPath);
```

Comment:
This is not async operation, so you can change the return type of the method to be `void` instead of `IFuture<void>` and remove the future wrapper.